packages/node/test/unit/install.spec.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  AppABIEncodings,
+  AppState,
+  AssetType,
+  BlockchainAsset
+} from "@counterfactual/types";
+import dotenv from "dotenv-extended";
+import { Wallet } from "ethers";
+import { AddressZero, One, Zero } from "ethers/constants";
+import { instance, mock, when } from "ts-mockito";
+import { v4 as generateUUID } from "uuid";
+
+import { ERRORS, install } from "../../src/methods/install-operations";
+import { openStateChannel } from "../../src/methods/multisig-operations";
+import { ProposedAppInstanceInfo } from "../../src/models";
+import { Store, STORE_ERRORS } from "../../src/store";
+import { EMPTY_NETWORK } from "../integration/utils";
+import MEMORY_STORE_SERVICE from "../services/memory-store-service";
```

Comment:
Does it make sense for this to be capitalized?

Reply:
Come to think of it, probably not. I'll camel case it.